inst/rmarkdown/templates/elsevier/skeleton/elsarticle.cls

```diff
@@ -5,36 +5,51 @@
 %% The original source files were:
```

Comment:
The problem is that many tex installations are outdated and the template won't work with older versions of `elsarticle.cls`. For example, the default installation of `texlive` on Ubuntu does not yet include this version of `elsarticle.cls`. So I thought we should just include it to avoid version hell for users.